Documentation/CommonDataModel_Wiki_Files/StandardizedClinicalDataTables/CONDITION_OCCURRENCE.md

```diff
@@ -15,7 +15,8 @@ Field|Required|Type|Description
 | condition_type_concept_id      | Yes       | integer      | A foreign key to the predefined Concept identifier in the Standardized Vocabularies reflecting the source data from which the condition was recorded, the level of standardization, and the type of occurrence.  |
 | stop_reason                    | No        | varchar(20)  | The reason that the condition was no longer present, as indicated in the source data.                                                                                                                            |
 | provider_id                    | No        | integer      | A foreign key to the Provider in the PROVIDER table who was responsible for capturing (diagnosing) the Condition.                                                                                                |
-| visit_occurrence_id            | No        | integer      | A foreign key to the visit in the VISIT_OCCURRENCE table during which the Condition was determined (diagnosed).                                                                                                             | visit_detail_id            | No        | integer      | A foreign key to the visit in the VISIT_DETAIL table during which the Condition was determined (diagnosed).                                                                                                             |
+| visit_occurrence_id            | No        | integer      | A foreign key to the visit in the VISIT_OCCURRENCE table during which the Condition was determined (diagnosed).                                                                                                  |
+| visit_detail_id                | No        | integer      | A foreign key to the visit in the VISIT_DETAIL table during which the Condition was determined (diagnosed).                                                                                                           | visit_detail_id            | No        | integer      | A foreign key to the visit in the VISIT_DETAIL table during which the Condition was determined (diagnosed).                                                                                                             |
```

Comment:
It seems that the visit_detail_id row is duplicated in the same line. Please remove one.

Reply:
@MaximMoinat I thought so too but when I looked at the rendered version of the table there is only one line for visit_detail_id